Builds/VisualStudio2015/libsoci_postgresql/libsoci_postgresql.vcxproj

```diff
@@ -166,7 +166,7 @@
       <GenerateDebugInformation>true</GenerateDebugInformation>
     </Link>
     <Lib>
-      <AdditionalDependencies>C:\Program Files\PostgreSQL\9.4\lib\libpq.lib</AdditionalDependencies>
+      <AdditionalDependencies>C:\Program Files\PostgreSQL\pg96\lib\libpq.a</AdditionalDependencies>
```

Comment:
You also need to change the install-windows.txt to reflect the change to PostgreSQL 9.6 if this was intended.